internal/flink/command_compute_pool_create.go

```diff
@@ -79,16 +79,18 @@ func (c *command) computePoolCreate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	computePoolEnv := computePool.Spec.GetEnvironment()
 	table := output.NewTable(cmd)
 	table.Add(&computePoolOut{
-		IsCurrent:  computePool.GetId() == c.Context.GetCurrentFlinkComputePool(),
-		Id:         computePool.GetId(),
-		Name:       computePool.Spec.GetDisplayName(),
-		CurrentCfu: computePool.Status.GetCurrentCfu(),
-		MaxCfu:     computePool.Spec.GetMaxCfu(),
-		Cloud:      computePool.Spec.GetCloud(),
-		Region:     computePool.Spec.GetRegion(),
-		Status:     computePool.Status.GetPhase(),
+		IsCurrent:   computePool.GetId() == c.Context.GetCurrentFlinkComputePool(),
+		Id:          computePool.GetId(),
+		Name:        computePool.Spec.GetDisplayName(),
+		Environment: computePoolEnv.GetId(),
```

Comment:
```suggestion
		Environment: computePool.Spec.Environment.GetId(),
```

Reply:
I was doing this to avoid nil pointers in case Environment was nil (even though that shouldn't actually happen)

Reply:
Yeah, shouldn't happen. We're planning on updating our SDK soon so we can handle scenarios where a breaking change to the API doesn't break the CLI